app/styles/_base.scss

```diff
@@ -98,8 +98,28 @@ a {
       top: auto;
     }
   }
+
+  :global &.-underline {
```

Comment:
I just :global because on json file can not add "className". That way we can use -underline class for two ways.